vllm/entrypoints/openai/api_server.py

```diff
@@ -418,7 +444,8 @@ async def create_completion(request: CompletionRequest, raw_request: Request):
             if len(request.prompt) > 1:
                 return create_error_response(
                     HTTPStatus.BAD_REQUEST,
-                    "multiple prompts in a batch is not currently supported")
+                    "multiple prompts in a batch is not currently supported",
+                )
```

Comment:
BTW are you using a different formatter from the `format.sh` in the project root directory? This PR includes many unnecessary style changes like this one. Before we merge, can you minimize these unnecessary style change? Thanks.

Reply:
I used the `format.sh`, which then introduces multiple changes... I suspect that it might result from some formatter version mismatch.